### PR TITLE
tests: update case settings for twr_ke18f boards

### PR DESCRIPTION
**Pull request by hakehuang**
disable MPU for cases exhaust MPU region on twr_ke18f
update ram size for benchmarks

Signed-off-by: Hake Huang <hake.huang@oss.nxp.com>

### CODE REVIEW

**Comment by henrikbrixandersen** [comment]
The KE1xF MPU region exhaustion is not limited to test cases. I submitted a few days back to address this issue in a more generic way: https://github.com/zephyrproject-rtos/zephyr/pull/31148

File: tests/benchmarks/sys_kernel/testcase.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 tests:
   benchmark.kernel.core:
     arch_exclude: nios2 riscv32 xtensa
-    min_ram: 32
+    min_ram: 36
```
Comment (henrikbrixandersen) [inline tests/benchmarks/sys_kernel/testcase.yaml:4]: I think this belongs in a separate commit in a separate PR.

**Review by henrikbrixandersen — CHANGES_REQUESTED** [review]

File: tests/benchmarks/sys_kernel/testcase.yaml (hunk 1 above)
Comment (hakehuang) [reply tests/benchmarks/sys_kernel/testcase.yaml:4]: @henrikbrixandersen  this case is not buildable in twr_ke18f, add this will skip this case in board testing.

Comment (henrikbrixandersen) [reply tests/benchmarks/sys_kernel/testcase.yaml:4]: > @henrikbrixandersen this case is not buildable in twr_ke18f, add this will skip this case in board testing.

I understand that. I still think it belongs in a separate commit with a dedicated commit log.

Comment (hakehuang) [reply tests/benchmarks/sys_kernel/testcase.yaml:4]: I see, let me drop the PR, and create another one
